recipe/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "torcheval" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: a498dec34137bc66c9cf1adc7353a46c604dd62255884c72dcb4e2e4fc2cd7e9
+    folder: pypi
+  - git_url: git@github.com:pytorch/torcheval.git
+    git_rev: f1bc22fc67ec2c77ee519aa4af8079f4fdaa41bb
+    git_depth: 1
+    folder: gitproj
```

Comment:
```suggestion
  # the upstream authors forgot *-requirements.txt, and the tests from the
  # sdist, here using the github archive for the specific commit id, because
  # they also forgot to add a tag for the release.
  url: https://github.com/pytorch/torcheval/archive/{{ commit_id }}.tar.gz
  sha256: 18f060f5152255635b168d1a09495d0e1c25f3418ba8fff32eeb172683c8244b
```

Reply:
I can use the GitHub archive but we should still prioritize pypi. That's why I'm using the GH tarball only to get the `requirements-*` files.